resources/assets/js/designer/BPMNHandler/Mutations.js

```diff
@@ -1,32 +1,77 @@
 import actions from "../actions"
-
-
+import _ from "lodash"
+/**
+ * Function to update participant object
+ * @param data
+ * @param elements
+ */
 function participantUpdate(data, elements) {
     if (elements[data.id] && data.bounds) {
         elements[data.id].diagram.elements[0].attributes = data.bounds
     }
 }
 
+/**
+ * Function to update shape
+ * @param data
+ * @param elements
+ */
 function shapeUpdate(data, elements) {
     if (elements[data.id] && data.bounds) {
         elements[data.id].diagram.elements[0].attributes = data.bounds
     }
 }
 
-function participantCreate(data, elements) {
-    let participant = {
+/**
+ * Function to create participant
+ * @param data
+ * @param elements
+ * @param arrayElements
+ * @param processes
+ * @param collaborations
+ */
+function participantCreate(data, elements, arrayElements, processes, collaborations) {
+    let process = {
         "type": "element",
         "name": "bpmn:participant",
-        "attributes": {"id": "Participant_18loy4z", "name": "Approval Link", "processRef": "approval_link"},
+        "attributes": {"id": data.id, "name": "Approval Link", "processRef": "approval_link"},
         "elements": []
     }
+
+    let diagram = {
+        "type": "element",
+        "name": "bpmndi:BPMNShape",
+        "attributes": {"id": data.id + "_di", "bpmnElement": data.id},
```

Comment:
Question..

Your are using "_di" suffix for this id but not in the previous element ( participant ), is that correct?